### PR TITLE
K1 Max Printer Definition

### PR DESCRIPTION
**Pull request by itay-grudev**
# Description

Adds printer definition for Creality K1 Max.

Please note that, when printing at High Speed, Cura won't enable the side fan which will result in poor cooling and deformation. Make sure to manually enable it before printing.

## Type of change

- [x] Printer definition file(s)

# How Has This Been Tested?

- [x] Tested at very high speed - 600mm/s.
- [x] Tested at high speed - 300mm/s.

**Test Configuration**:
* Operating System: Windows / Linux

# Checklist:
- [x] My code follows the style guidelines of this project as described in [UltiMaker Meta](https://github.com/Ultimaker/Meta) and [Cura QML best practices](https://github.com/Ultimaker/Cura/wiki/QML-Best-Practices)
- [x] I have read the [Contribution guide](https://github.com/Ultimaker/Cura/blob/main/CONTRIBUTING.md) 
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have uploaded any files required to test this change

Fixes: #17417

### CODE REVIEW

File: resources/definitions/creality_k1max.def.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        "machine_width":                    { "default_value": 300 },
+        "retraction_amount":                { "default_value": 0.5 },
+        "retraction_speed":                 { "default_value": 40 },
+        "speed_layer_0":                    { "default_value": 60 },
+        "speed_print":                      { "default_value": 300 },
+        "speed_travel":                     { "default_value": 500 },
+        "retraction_min_travel":            { "default_value": 2 },
+        "retraction_hop":                   { "default_value": 0.1 },
+        "retraction_combing_max_distance":  { "default_value": 15 },
+        "material_diameter":                { "value": 1.75 },
```
[the remark this answers is not in the document]
Comment (itay-grudev) [reply resources/definitions/creality_k1max.def.json]: Done.

**Comment by ccdunder** [comment]
LGTM

I came up with the same values independently and can confirm they work well with the K1 Max.

I'm guessing we took the same approach: pull values from Creality Print (reskinned Cura) and refine them (e.g. measure the correct machine head and gantry dimensions).

Nice work!

**Comment by evanbarn** [comment]
What needs to be done to get this PR merged into main?

**Comment by saumyaj3** [comment]
Hey @itay-grudev ! Thanks for the contribution. I can not test the profiles because I don't have the printer. But I believe community is satisfied with it. (y) LGTM.

**Comment by saumyaj3** [comment]
I created https://github.com/Ultimaker/Cura/commit/003808c9a252fb933eb0cd238560ca8f608b787b commit to fix the unit tests. 
I removed those default values as they won't change anything. creality_base has values  in those fields. if you wish to add these values again, please create another PR referencing this PR, with default_values as "values"

**Comment by carribeiro** [comment]
I'm testing it now, starting slow (150mm/s). I'll let you know how it goes. I've noticed that it is noticeably slower that the same file sliced in Orca with the same speed, so I guess there's still some work to fine tune the acceleration.

**Comment by carribeiro** [comment]
Jus one note - the print actually took about the same time that it took with Orca. The time estimate is really off, I don't know why.

**Comment by ccdunder** [comment]
K1 uses Klipper so something like klipper_estimator is needed for accurate
estimates.

https://github.com/Annex-Engineering/klipper_estimator

**Pull request merged**